package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "nva-frontend",
+  "type": "module",
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "react-scripts build",
+    "build": "tsc && vite build",
     "pre-commit": "npx lint-staged && npm run unused-i18n-keys",
     "prepare": "husky",
-    "start": "react-scripts -r @cypress/instrument-cra start",
-    "start:mock": "set REACT_APP_USE_MOCK=true&& npm start",
+    "preview": "vite preview",
+    "start": "vite",
+    "start:mock": "set VITE_USE_MOCK=true&& npm start",
     "test": "cypress run",
     "test:cypress": "start-server-and-test start:mock http://localhost:3000 test",
```

Comment:
tror denne og/eller `start:mock` må justeres litt for at testene skal kjøre som forventet

Reply:
Ser ut som det funka å "bloate" buildspecen enda mer, med å installere xdg-utils før testene funka. lever ok med den løsningen.